dotcom-rendering/src/components/Discussion/Comments.tsx

```diff
@@ -388,7 +414,10 @@ export const Comments = ({
 			) : !comments.length ? (
 				<NoComments />
 			) : (
-				<ul css={commentContainerStyles}>
+				<ul
+					css={[commentContainerStyles, labelStyles]}
+					className="comments-column"
```

Comment:
As mentioned in the commercial PR, a data attribute is perhaps more appropriate here instead of a className

Reply:
Yup done that in both repos